pkg/restapi/vc/operation/operations_test.go

```diff
@@ -37,14 +43,13 @@ import (
 	"github.com/trustbloc/edge-service/pkg/internal/mock/edv"
 )
 
-const multipleContext = `"@context":["https://www.w3.org/2018/credentials/v1","https://w3id.org/citizenship/v1"]`
-
-const validContext = `"@context":["https://www.w3.org/2018/credentials/v1"]`
-
-const invalidContext = `"@context":"https://www.w3.org/2018/credentials/v1"`
+const (
```

Comment:
I took all the different `const` declarations and grouped them all, so that's why you'll see so many line changes around here.